src/submission/schema/schema-manager.ts

```diff
@@ -222,22 +235,25 @@ namespace MigrationManager {
       throw new Error('failed to submit migration');
     }
 
-    if (dryRun) {
-      const result = await startMigration(savedMigration);
-      return result;
-    }
+    return await runMigrationSyncOrAsync(sync || savedMigration.dryRun, savedMigration);
+  };
 
+  async function runMigrationSyncOrAsync(
```

Comment:
no need to say `SyncOrAsync` in the name

Reply:
hmm I only named it that since I renamed `startMigration` to `runMigration`, since it doesn't just start migration...

Reply:
no the new name is ok without the SyncOrAsync.. ie. `runMigration(...)`

Reply:
I see what you are saying